plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/RegisterTableProcedure.java

```diff
@@ -195,6 +195,8 @@ private static String normalizeS3Uri(String tableLocation)
     {
```

Comment:
"Register procedure should accept locations which differ with trailing slash" -> 
"Strip trailing slash in `register_table` from the table location parameter"